### PR TITLE
implemented breadcrumbs

### PR DESCRIPTION
**Pull request by SergeyKoliadintsev**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201691636463526

### CODE REVIEW

**Comment by VladyslavBanul** [comment]
The breadcrumb component is too wide and it unnecessarily stretch page

File: src/storage/singletons/Breadcrumbs/index.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { makeAutoObservable, runInAction } from "mobx";
+import Tenant from "../Tenant";
+
+class BreadcrumbsStorage {
+  customerLevels: any = [];
```
Comment (viktorsmakhtin) [inline src/storage/singletons/Breadcrumbs/index.ts:5]: pls provide a type { name: string; disabled: boolean; link: "string" }[]

File: src/storage/singletons/Breadcrumbs/index.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  isLoading: boolean = false;
+
+  constructor() {
+    makeAutoObservable(this, {});
+  }
+
+  setCustomerLevel = async (payload: Array<any>) => {
```
Comment (viktorsmakhtin) [inline src/storage/singletons/Breadcrumbs/index.ts:12]: provide a type for payload

File: src/storage/singletons/Tenant/index.ts [hunk 3]
```diff
@@ -108,6 +108,19 @@ class TenantStore {
     return result!.data;
   };
 
+  getSpecificTenantSubscription = async (
+    tenantID: string,
+    subscriptionID: string,
+  ): Promise<any | undefined> => {
```
Comment (viktorsmakhtin) [inline src/storage/singletons/Tenant/index.ts:114]: pls provide a type

File: src/components/Breadcrumbs/index.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    location: { pathname },
+  } = history;
+
+  const {
+    customerLevels,
+    cleanBreadcrambsStorage,
+    isLoading,
+  } = BreadcrumbsStorage;
+
+  const pathnames: any = pathname.split("/").filter(x => x);
```
Comment (VladyslavBanul) [inline src/components/Breadcrumbs/index.tsx:27]: Add type there

File: src/components/Breadcrumbs/index.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+  return (
+    <>
+      {(customerLevels.length > 0 && !isLoading) || !isLoading ? (
+        <MUIBreadcrumbs
+          separator={separator}
+          className={classes.breadcrumbsWrapper}
+          aria-label="breadcrumb"
+        >
+          {customerLevels.map((item: any, index: any) => {
```
Comment (VladyslavBanul) [inline src/components/Breadcrumbs/index.tsx:45]: Add types there

**Pull request merged**